src/freenet/pluginmanager/OfficialPlugins.java

```diff
@@ -262,6 +274,7 @@ private OfficialPluginDescription createOfficialPluginDescription() {
 			this.deprecated = deprecated;
 			this.experimental = experimental;
 			this.advanced = advanced;
+      this.unsupported = unsupported;
```

Comment:
Now that I've typed it I'm not so sure that makes sense - why be out of line with existing code if we're going to autoconvert? That said there's so much code that has yet to be merged that it may never be reasonable to autoconvert.